src/Unicorn.Tests/Predicates/TestConfiguration.xml

```diff
@@ -25,6 +25,17 @@
 		<exclude path="/implicit-nochildren/" /> <!-- implicit exclusion of a child path. NOTE: does not work for paths under the root (e.g. /implicit-nochildren/foo/) -->
 	</include>
 
+  <!-- 
+		TEMPLATE-ID: Exclude all children with a given templateId		
```

Comment:
Add a note that children of items with the excluded template ID will also be excluded